tests/test_unisender_go_payload.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+from django.test import SimpleTestCase, override_settings
+
+from anymail.backends.unisender_go import EmailBackend, UnisenderGoPayload
+from anymail.message import AnymailMessageMixin
+
+TEMPLATE_ID = "template_id"
+FROM_EMAIL = "sender@test.test"
+FROM_NAME = "test name"
+TO_EMAIL = "receiver@test.test"
+TO_NAME = "receiver"
+OTHER_TO_EMAIL = "receiver1@test.test"
+OTHER_TO_NAME = "receiver1"
+SUBJECT = "subject"
+GLOBAL_DATA = {"arg": "arg"}
+SUBSTITUTION_ONE = {"arg1": "arg1"}
+SUBSTITUTION_TWO = {"arg2": "arg2"}
+
+
+class TestUnisenderGoPayload(SimpleTestCase):
+    @override_settings(
+        ANYMAIL_UNISENDERGO_SKIP_UNSUBSCRIBE=False,
+        ANYMAIL_UNISENDERGO_API_KEY=None,
+        ANYMAIL_UNISENDERGO_API_URL="",
+    )
```

Comment:
(You can apply `@override_settings` at the class level, if every test in the class wants these overrides.)